### ISSUE TITLE
Contributing: Can't initialize Xcode project

### ISSUE THREAD

**Issue by relikd** [Carthage/Carthage#2839]
I've cloned this repository and did run `make clean && SWIFTPM_BOOTSTRAP=true make xcodeproj`. All dependencies get resolved and the Xcode project file is created. However, if I open the Xcode project and try to build it, `SPMLLBuild/llbuild.swift` will fail with:

```
@_exported import llbuildSwift         <--- Missing required module 'llbuild'
```

**Comment by relikd** [comment]
Updating the `swift-package-manager` dependency to `swift-4.2-branch` will fix these build issues and the project can be run. However, in doing so, other issues arise when building the release version with `make install`.

**Comment by olejnjak** [comment]
Yep, same issue here, wanted to try to fix #2844 but unable to build the project from `master` branch.

**Comment by tmspzz** [comment]
That's curious. I don't have a solution for this at the moment.

Off the top of my head I would probably add to the search paths the location of `llbuild` (see .travis.yml)

**Comment by relikd** [comment]
Tried five different search paths, no luck.

My workaround so far:

1. Change the branch to 4.2 and update all `SPMUtility` header files (as seen [here](https://github.com/Carthage/Carthage/pull/2838/files)).
2. Use this environment for development (it compiles nicely in Xcode)
3. Copy all changes manually to the `master` branch (without the 4.2 branch modifications)
4. Run `make clean && SWIFTPM_BOOTSTRAP=true make test` 
5. Run `bats IntegrationTests`

**Comment by manicmaniac** [comment]
I'm facing the same issue after posting https://github.com/Carthage/Carthage/pull/2853.

**Comment by jdhealy** [comment]
Fixed in #2863.
